.env.example

```diff
@@ -24,8 +24,8 @@ TUNNEL_SUBDOMAIN=<subdomain>
 PRIVATE_KEY_PATH=<your-private-key>.pem
 ATLASSIAN_URL=https://<your-instance>.atlassian.net
 TRACKING_DISABLED=true
-HYDRO_BASE_URL=https://hydro-base-url.com/api/v1/events
-HYDRO_APP_SECRET=2dd220c366ec5b86104efd7324c2d405
+HYDRO_BASE_URL=
+HYDRO_APP_SECRET=
```

Comment:
I've removed this from our code, hence we don't need it anyway

Reply:
We can remove the lines completely!